lib/solarwinds_apm/api/current_trace_info.rb

```diff
@@ -98,8 +98,7 @@ def for_log
         # * Hash
         #
         def hash_for_log
-          @hash_for_log = {}
-          @hash_for_log = {trace_id: @trace_id, span_id: @span_id, trace_flags: @trace_flags, service_name: @service_name} if @do_log
+          @hash_for_log = @do_log ? {'trace_id' => @trace_id, 'span_id' => @span_id, 'trace_flags' => @trace_flags, 'resource.service.name' => @service_name} : {}
```

Comment:
Nice, so this also fixes the minor problem you mentioned, that we were providing service name as `service_name` instead of `resource.service.name`, right?

Reply:
Yes, it fixed that issue.